discord/interactions/addConfigCommand.ts

```diff
@@ -288,7 +288,7 @@ export const handleConfigConfirmCommand = async (
       discordServerId: interaction.guildId,
       starkyModuleConfig: currentConfig.moduleConfig,
       discordRoleId: currentConfig.roleId,
-      starknetNetwork: currentConfig.network,
+      //starknetNetwork: currentConfig.network,
```

Comment:
Why ? And don't comment code you don't use anymore, just delete. Else we will end up with comments everywhere